library/ic-ledger-types/src/lib.rs

```diff
@@ -30,6 +41,7 @@ pub const MAINNET_CYCLES_MINTING_CANISTER_ID: Principal =
     CandidType, Serialize, Deserialize, Clone, Copy, Hash, Debug, PartialEq, Eq, PartialOrd, Ord,
 )]
 pub struct Timestamp {
+    /// The nanosecond count.
```

Comment:
See https://github.com/dfinity/ic/blob/master/rs/rosetta-api/icp_ledger/ledger.did#L8

```suggestion
    /// Number of nanoseconds from the UNIX epoch in UTC timezone.
```

Reply:
I assumed there'd be no point in repeating that from the struct docs, but :shrug: